modules/security/src/main/Permission.scala

```diff
@@ -75,6 +77,17 @@ object Permission {
         "Lichess team"
       )
 
+  case object TimeoutMod
+      extends Permission(
+        "TIMEOUT_MOD",
+        List(
+          ChatTimeout,
+          PublicChatView,
+          GamifyView
```

Comment:
so this means that all timeout mods can now see the full list of all other mods?